Hometask_2/Hometask_2.py

```diff
@@ -0,0 +1,18 @@
+from random import randint, choice  # importing module
+from string import ascii_lowercase  # importing module
+final_dict, temp_dict = {},  {}  # create a dictionaries
```

Comment:
Just for your information:
According to PEP8, it's not the best practice to use variable type in variable name ('dict', 'list' and so on).
But we can consider that we have an exceptional case.